### ISSUE TITLE
[Feat]: add a chart with dbengine metrics retention time (for all tiers) to Netdata Monitoring

### ISSUE THREAD

**Issue by ilyam8** [netdata/netdata#14798]
### Problem

Science degree is needed to [achieve desired metrics retention time](https://learn.netdata.cloud/docs/deployment-in-production/metric-retention---database/change-how-long-netdata-stores-metrics#calculate-the-system-resources-ram-disk-space-needed-to-store-metrics) 🤓

In theory, you _just_ need to know the number of concurrently collected metrics to configure "dbengine multihost disk space MB" for all tiers. In practice, if we don't count the most simplistic setups (e.g. a standalone instance with no containers), and consider a host with ephemeral containers (e.g. k8s worker node) or a parent instance - it is close to impossible.

There is really no way to know the actual metrics retention time!

### Description

Add a chart that shows the **actual** retention time based on current settings and currently collected metrics (and any other factors that contribute to this).

**It would be nice to have a chart with tierN as dimensions and the actual retention time as values.**

### Importance

must have

### Value proposition

1. Being able to see the actual metrics retention time based on the current workload.
2. If some retention time is required, we will be able to create alarms (e.g. tier0 retention time < 5 days).

**Comment by stelfrag** [comment]
Something like this? 

![image](https://user-images.githubusercontent.com/52996999/233346022-5587d8f0-9e34-4d01-bade-0890e2c0622b.png)

**Comment by ilyam8** [comment]
@stelfrag exactly!

**Comment by ilyam8** [comment]
Both the current and expected retention time per tier statistics are added in #14798 (https://github.com/netdata/netdata/pull/15294/commits/c301ac4ea5939724fcf9813699795e2cb350c993)

**Comment by luisj1983** [comment]
I'd add the following points...

What people usually want to know is:

1. How far into the past does my data go?
2. How utilized are the resources I've assigned for my dbengine?
- how do I know that I've assigned enough resources, too little or _too much_?

Killer feature: a chart which provides a prediction-cone-into-the-future of estimated future usage based on growth-rate capped by settings.
Failing that, having this stuff as tile or badge-style.

It's worth considering as a separate Feature Request that the previous db engine calculator page should be updated and, if possible, actually added into the cloud UI.

**Comment by nodiscc** [comment]
related https://github.com/netdata/netdata/issues/17082 - it would remove the need to do correct, manual database size calculations, which are error-prone in even moderately complex setups, hence this chart would not be needed anymore.

**Comment by ilyam8** [comment]
@nodiscc time-based retention is #17413

**Comment by ilyam8** [comment]
Closing because not really relevant after #17413
